### ISSUE TITLE
Trying to delete nodes in required relations doesn't give feedback

### ISSUE THREAD

**Issue by marktani** [graphcool/prisma-cloud-feedback#127]
If you try to delete a node that is in a required relation, and deleting it would violate that constraint, you don't get any feedback about this.

Click "Delete Node"

![image](https://user-images.githubusercontent.com/1780597/38408067-3ba84456-397c-11e8-8869-a272505b4289.png)

This error message is hidden:

![image](https://user-images.githubusercontent.com/1780597/38408029-1a241882-397c-11e8-8093-0750771005a3.png)

Instead, the error message should be shown in the top right corner of the Console.

**Comment by kuldar** [comment]
We need a confirmation view for the node deletion flow anyway (#163), so the deletion failure feedback could fit into the same format.

![node deletion](https://user-images.githubusercontent.com/1710629/42889452-c244cc80-8aaa-11e8-86e0-f07052d965cf.png)

**Comment by marktani** [comment]
Looks great! I think it would be helpful to include specific information about the node. At least the `id`, but also other fields. This helps me to decide whether I really want to delete the node, or detect that I misclicked on the wrong row.

**Comment by kuldar** [comment]
Yeah, I tried coming up with some reference info as well, but the type name was best I could come up with.

I don't think anyone would really recognize the nodes by IDs and as for all the other fields, it's pretty much unknown which fields would give meaningful recognition to the end user.
